docs/docs/examples/guides/object-detection-example.md

```diff
@@ -1,91 +1,199 @@
 ---
-sidebar_position: 8
+sidebar_position: 14
 title: Using YOLO to track objects
-unlisted: true
 ---
 
-import rvizgif from './assets/object-detection-example-rviz.gif'
+import exampleApp from './assets/object-detection-example-app.gif'
+import yoloExecutor from './assets/object-detection-yolo-executor.png'
+import yoloExecutorParameters from './assets/object-detection-yolo-executor-parameters.png'
+import boundingBoxTracker from './assets/object-detection-robot-control.png'
+import launcherToolkitsCPU from './assets/launcher-toolkits-cpu.png'
+import launcherToolkitsGPU from './assets/launcher-toolkits-gpu.png'
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
 
 # Using YOLO to track objects
 
-<!-- TODO: replace `Object Detection Components` with whatever the package name would be in Launcher -->
-This example provides a use case for `Object Detection Components`. We show how to create a custom component which
-converts bounding boxes into a 3D pose. The example works with any RBG `sensor_msgs::msg::Image` signal, such as
-provided by the Camera Streamer component from `components/core-vision`. The final application will be able to track an
-object with a robot based on a fixed camera position.
+YOLO (You Only Look Once) is a real-time object detection algorithm that predicts bounding boxes and class probabilities
+directly from an image in a single pass through a neural network. Unlike older methods that scan an image multiple times,
+YOLO processes the entire image at once, making it extremely fast and well-suited for many applications, including
+robotics.
+
+## A YOLO example using the AICA framework
+
+This page details how to run a `YoloExecutor` component, i.e., a component that can use various YOLO models for
+inference. It also demonstrates how it could be used as part of an AICA application. In following paragraphs, we show how to create a custom
```

Comment:
```suggestion
inference. It also demonstrates how it could be used as part of an AICA application. In the following paragraphs, we show how to create a custom
```
You also have a bunch of long lines here